Cargo.toml

```diff
@@ -12,24 +12,24 @@ rust-version = "1.62"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["blocking", "asynk"]
+default = ["blocking"]
```

Comment:
why did you remove "asynk"?

Reply:
For some reason you deleted in your commit i will put again 


![image](https://user-images.githubusercontent.com/48651252/187252431-71a4295d-01cb-4048-8d66-24e953aad082.png)

Reply:
[asynk feature](https://github.com/ayrat555/fang/pull/74/commits/44babbcea2b98cf2aa241b2d3d249bedf01f6e21)